enterprise/backend/src/metabase_enterprise/dependencies/task/entity_check.clj

```diff
@@ -0,0 +1,72 @@
+(ns metabase-enterprise.dependencies.task.entity-check
+  (:require
+   [clojurewerkz.quartzite.jobs :as jobs]
+   [clojurewerkz.quartzite.scheduler :as qs]
+   [clojurewerkz.quartzite.triggers :as triggers]
+   [java-time.api :as t]
+   [metabase-enterprise.dependencies.findings :as deps.findings]
+   [metabase-enterprise.dependencies.settings :as deps.settings]
+   [metabase.premium-features.core :as premium-features]
+   [metabase.task.core :as task]
+   [metabase.util.log :as log]))
+
+(set! *warn-on-reflection* true)
+
+(defn- current-millis
+  "Returns the current epoch millis. Uses java-time so that clock settings can be used in tests."
+  []
+  (t/to-millis-from-epoch (t/instant)))
+
+(def ^:private entities
+  [:card :transform :segment])
```

Comment:
I feel like I've seen this group of three entities is defined all over the place in this module, it would be really nice if it could just be defined once, maybe in the module `.schema` namespace or something, so adding support for a new model didn't require updating 100 things